tests/tests.c

```diff
@@ -62,6 +62,7 @@ static sir_test sir_tests[] = {
     {"filesystem",              sirtest_filesystem, false, true},
     {"squelch-spam",            sirtest_squelchspam, false, true},
     {"plugin-loader",           sirtest_pluginloader, false, true},
+    {"strutils",                sirtest_strutils, false, true},
```

Comment:
recommend the pattern we've kind of established: `word-anotherword`. maybe `string-utils` or if you choose `strmanip` for the filename, `string-manip`

Reply:
I'll go with string-utils/stringutils